src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -41,17 +107,23 @@ public void keyServerSetup(String keyServerIp, String keyServerHttpPort) {
     this.keyServerHttpPort = keyServerHttpPort;
   }
 
+  /**
+   * Setup the server with use.sh
```

Comment:
Period is missing at the end of the sentence and the sentence doesn't start with an action, i. e. "Sets up the server with use.sh."